test/language/literals/numeric/numeric-separator-literal-nzd-nsl-dd-one-of.js

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2017 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: prod-NumericLiteralSeparator
+description: NonZeroDigit NumericLiteralSeparator DecimalDigit
+info: >
+  NumericLiteralSeparator ::
+    _
+
+  DecimalIntegerLiteral ::
+    ...
+    NonZeroDigit NumericLiteralSeparator_opt DecimalDigits
+
+  NonZeroDigit :: one of
+    1 2 3 4 5 6 7 8 9
+
+  DecimalDigits ::
+    DecimalDigit
+    ...
+
+  DecimalDigit :: one of
+    0 1 2 3 4 5 6 7 8 9
+
+---*/
+
+assert.sameValue(1_0, 10);
```

Comment:
this case already exists in other files. I don't mind, but if the idea is testing one of, it would nice to add more assertions in this same file for:

2_1, 2_3, 2_4...
3_1, 3_2, 3_4...
...